diff -ur emelfm2-0.7.1.orig/plugins/optional/gimpthumb.c emelfm2-0.7.1/plugins/optional/gimpthumb.c
--- emelfm2-0.7.1.orig/plugins/optional/gimpthumb.c	2009-10-15 03:26:19.000000000 +0200
+++ emelfm2-0.7.1/plugins/optional/gimpthumb.c	2009-12-30 14:51:33.000000000 +0100
@@ -869,7 +869,7 @@
   if (thumbnail->image_mimetype)
     {
       keys[i]   = TAG_THUMB_MIMETYPE;
-      values[i] = g_strdup_printf (thumbnail->image_mimetype);
+      values[i] = g_strdup_printf ("%s", thumbnail->image_mimetype);
       i++;
     }
 
Only in emelfm2-0.7.1/plugins/optional: gimpthumb.c~
